packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -49,6 +49,12 @@ export const PRESET_METADATA = [
 			},
 		],
 	},
+	{
+		path: [ 'color', 'duotone' ],
+		cssVarInfix: 'duotone',
+		valueFunc: ( { slug } ) => `url( '#wp-duotone-${ slug }' )`,
```

Comment:
I'm a bit unsure about this. It would be nice to specify it in duotone.js, but adding a hook for it seems a bit like overkill.

Reply:
Thinking of what happens when we allow users to be able to change the duotone assigned to a block via the global styles sidebar: at that point, we'd need the ability to regenerate the styles in the client, hence, we need this code. I think this should be fine, it works the same as any other existing preset (server & client implementation).

Reply:
> Thinking of what happens when we allow users to be able to change the duotone assigned to a block via the global styles sidebar: at that point, we'd need the ability to regenerate the styles in the client, hence, we need this code.

Wouldn't that mean we'll need the ability to regenerate the SVGs too on the frontend then?